instagram/instagram/src/components/PostContainer.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./postcontainer.css";
+import message from "../message.png";
+import heart from "../heart.png";
+import CommentContainer from "./CommentContainer";
+
+const PostContainer = props => {
+  console.log(props.dummydata[0].username);
+  return (
+    <div className="postcontainer">
+      <div className="posttitle">
+        {props.dummydata.map(person => (
```

Comment:
ideally we map from app.js and create a `<PostContainer/>` for _each_ post, instead of mapping in PostContainer. an easy refactor